### ISSUE TITLE
Handlers dictionary

### ISSUE THREAD

**Issue by urosjarc** [pfalcon/npyscreen#27]
I have hard time to figure it out how can I add_handlers for enter or space.
For example this don't works for me...

    self.add_handlers({
        curses.KEY_ENTER : self.updateValues
    })

But this works...

    self.add_handlers({
        'a' : self.updateValues
    })

I had try your example but still its a fail...

    curses.KEY_UP:     self.h_exit_up,

Is it me doing something wrong?

**Comment by pfalcon** [comment]
> Is it me doing something wrong?

Yes, you're submitting support requests against random fork of a project. Please use upstream project: https://code.google.com/p/npyscreen/ . And tell its author that he should migrate to github ;-).

**Comment by urosjarc** [comment]
I'm putting my solution for this problem here because I was been asked if I have solved the problem...

	npyscreen.add_test_input_ch(ord(' '))     #This is space key...
	npyscreen.add_test_input_ch(ord('\n'))    #This is enter key...
